lib/msal-common/docs/Accounts.md

```diff
@@ -0,0 +1,62 @@
+# Accounts in MSAL Javascript
+
+MSAL Javascript libraries (`msal-browser` and `msal-node`) support both single account and multiple accounts scenarios in javascript applications. An `account` object is standardized across the libraries:
+
+```javascript
+export type AccountInfo = {
+    homeAccountId: string;
+    environment: string;
+    tenantId: string;
+    username: string;
+};
+```
+
+## Usage
+
+* We provide a public API `getAllAccounts()` which lists all the accounts currently in the cache. An application must choose an account to acquire tokens silently.
+* `msal-browser` provides a public API `getAccountByUsername()` for an application to retrieve a specific account based on `username`
+
+Sample usage is as below:
+
+``` javascript
+
+function handleResponse(resp) {
+    if (resp !== null) {
+        username = resp.account.username;
+        ...
+    } else {
+        // need to call getAccount here?
+        const currentAccounts = myMSALObj.getAllAccounts();
+        if (currentAccounts === null) {
+            return;
+        } else if (currentAccounts.length > 1) {
+            // Add choose account code here
```

Comment:
Yeah, my only call out that I brought up in the other doc was this here. I think we should provide a baseline multiple accounts case that we can point people to--either here or in a "multiple accounts" sample

Reply:
Agreed, we will work to add some samples showing multiple account scenarios in future PRs.